pkg/plugins/platform/local.go

```diff
@@ -801,20 +806,18 @@ func (l *LocalApp) CreateSettingsFile() error {
 	return nil
 }
 
-// Down stops the docker containers for the local project.
+// Down stops the docker containers for the project in current directory.
 func (l *LocalApp) Down(removeData bool) error {
 	l.DockerEnv()
 	settingsFilePath := l.AppConfig.SiteSettingsPath
 
-	_, _, err := dockerutil.ComposeCmd(l.ComposeFiles(), "down", "-v")
+	// Remove all the containers and volumes for app.
+	err := Cleanup(l)
```

Comment:
A container not having a `com.ddev.site-name` label. This might occur if a user added a third-party container (Such as solr) or modified a docker-compose file in an ill-advised way.

I think we're probably OK with that small risk, but it should be pointed out.

Reply:
Isn't that something we've insisted on in the add-on instructions, that they do that? It's been a long time.

Reply:
I believe we do, yes. There's always some danger in relying on users to do anything, though. I'm totally fine with the change, was just noting that it's not completely without risk.